### DISCUSSION TITLE
所有链接都无法下载

### DISCUSSION THREAD

**Discussion by whiteS18** [YerongAI/Office-Tool#728] [Q&A] [answered]
在官网的下载页面几个下载渠道都无法访问了，打开之后没反应，开启关闭梯子都不行

**Comment by whiteS18** [comment]
补充标签截图...
![图片](https://github.com/YerongAI/Office-Tool/assets/31566522/b4e2562b-5b3a-4e72-ba33-47f9e2c7a419)

**Reply by YerongAI** [reply, marked as answer]
现在可以了吗

**Reply by whiteS18** [reply]
可以了
